LethalAPI.Core/Patches/GameNetworkManagerJoinPatch.cs

```diff
@@ -86,15 +96,15 @@ private static bool LobbyDataIsJoinable_Prefix(GameNetworkManager __instance, re
             return false;
         }
 
-        if (lobby.MemberCount >= 4 || lobby.MemberCount < 1)
+        if (lobby.MemberCount is >= 4 or < 1)
```

Comment:
What is this godly line?. I don't think we should do that. We should revert change on this line

Reply:
Should be fine. Modern CSharp is, well, modern.

Reply:
Its a language 11 feature. Anyone working on the core api should have the latest minor language version. The CSProj file will ensure it is enabled. 

I think it is important to use new features, as it speeds up development, and cleans up the code. Developers can still interpret this but it takes up less space. In some cases `is` is actually more efficient and faster than `==`